s3/pkg/service/service.go

```diff
@@ -541,6 +541,32 @@ func (s *s3Service) CountObjects(ctx context.Context, in *pb.ListObjectsRequest,
 	return nil
 }
 
+func (s *s3Service) BackendCheck(ctx context.Context, backendDetail *pb.BackendDetailS3, out *pb.BaseResponse) error {
+	log.Info("backendCheck is called in s3 service.")
```

Comment:
Make the Info messages started with Upper case

Reply:
done